conjure-java-client-verifier/src/test/java/com/palantir/conjure/java/compliance/VerificationClients.java

```diff
@@ -92,7 +93,24 @@ public static SingleQueryParamService singleQueryParamService(VerificationServer
     }
 
     public static SingleQueryParamServiceBlocking dialogueSingleQueryParamService(VerificationServerRule server) {
-        return SingleQueryParamServiceBlocking.of(
-                JavaChannels.create(server.getClientConfiguration()), DEFAULT_CONJURE_RUNTIME);
+        return dialogue(SingleQueryParamServiceBlocking.class, server);
+    }
+
+    private static <T> T dialogue(Class<T> clazz, VerificationServerRule server) {
+        ClientConfiguration config = server.getClientConfiguration();
+        return DialogueClients.create(Refreshable.only(ServicesConfigBlock.builder()
+                        .putServices(
+                                "service",
+                                PartialServiceConfiguration.builder()
+                                        .uris(server.getClientConfiguration().uris())
+                                        .build())
+                        .defaultSecurity(server.getSslConfiguration())
+                        .build()))
+                .withUserAgent(config.userAgent().orElseThrow(AssertionError::new))
+                .withTaggedMetrics(config.taggedMetricRegistry())
+                .withClientQoS(config.clientQoS())
+                .withServerQoS(config.serverQoS())
+                .withRuntime(DEFAULT_CONJURE_RUNTIME)
+                .get(clazz, "service");
```

Comment:
would also be fine to use the getNonReloading method here I think?

Reply:
Can do that, it's a tad odd given the refreshable required to create the initial factory, but that's probably fine

Reply:
I tried it out, it will require this change to pass tests: https://github.com/palantir/dialogue/pull/744